### DISCUSSION TITLE
Show mimetype detected by yazi

### DISCUSSION THREAD

**Discussion by raffaelemancuso** [sxyazi/yazi#1015] [1. Q&A]
The `file` command detects my neovim configuration files as `text/plain` at best and `application/javascript` at worst.

While the correct mime is `text/x-lua`.

I installed [mime.yazi](https://github.com/DreamMaoMao/mime.yazi) plugin.

How can I show the mimetype detected by yazi, to be sure it works?

(it __seems__ to work as now they are opened by the `text/*` opener)

**Comment by sxyazi** [comment]
There's a FR about it "A floating window to glance at more information about this file (mime-type, atime/ctime/mtime, previewer, preloaders, EXIF for images/videos, etc.)" in the [Feature Requests](https://github.com/sxyazi/yazi/issues/51), I haven't found the time to implement it

**Comment by sxyazi** [comment]
Now supported in https://github.com/sxyazi/yazi/pull/1802
